CMakeLists.txt

```diff
@@ -2091,7 +2092,7 @@ elseif(WIN32)
       # QtNetwork openssl-linked
       crypt32
 
-      # New libraries required by Qt5.
+      # New libraries required by Qt${QT_VERSION_MAJOR}.
```

Comment:
Please keep the original "Qt5" string. Replacing it with the template is wrong.

Reply:
good catch

Reply:
Some of those comments don't provide any value. If they no longer apply please delete or rephrase them. Remainders from the Qt4/Qt5 migration.